server/sonar-main/src/main/java/org/sonar/application/process/ProcessLauncherImpl.java

```diff
@@ -153,7 +166,7 @@ public ProcessMonitor launch(JavaCommand javaCommand) {
 
   private ProcessBuilder create(EsCommand esCommand) {
     List<String> commands = new ArrayList<>();
-    commands.add(esCommand.getFileSystem().getExecutable().getAbsolutePath());
+    commands.add(esCommand.getElasticsearchConfiguration().getExecutable().getAbsolutePath());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "getElasticsearchConfiguration" is nullable here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2259)